lib/internal/fs/promises.js

```diff
@@ -917,6 +919,26 @@ async function readFile(path, options) {
   return handleFdClose(readFileHandle(fd, options), fd.close);
 }
 
+async function* _watch(filename, options = kEmptyObject) {
+  validateObject(options, 'options');
+
+  if (options.recursive != null) {
+    validateBoolean(options.recursive, 'options.recursive');
+
+    // TODO(anonrig): Remove this when/if libuv supports it.
+    // libuv does not support recursive file watch on all platforms, e.g. Linux due to
+    // the limitations of inotify.
```

Comment:
Optional/non-blocking suggestion: Put a November 2022 and/or the current-as-of-this-writing libuv version in this comment.

Reply:
Doesn't git blame provide the necessary information for this?

Reply:
It does. My suggestion is just an optional convenience for people reading the code later so they don't have to do that. (And it won't be made more annoying/difficult if people come in and edit the commit message. We do get PRs on spelling fixes in code comments, for example.)